consensus/cometbft/service/service.go

```diff
@@ -102,9 +112,12 @@ func NewService[
 			db,
 			servercmtlog.WrapSDKLogger(logger),
 		),
-		Middleware: middleware,
-		cmtCfg:     cmtCfg,
-		paramStore: params.NewConsensusParamsStore(cs),
+		Blockchain:    blockchain,
+		BlockBuilder:  blockBuilder,
+		cmtCfg:        cmtCfg,
+		chainSpec:     cs,
```

Comment:
@fridrik01 seems to me you don't need to store `chainSpec` here. This is just used to create paramStore